index.html

```diff
@@ -97,6 +97,35 @@ <h2>Connect</h2>
           <a href="www.linkedin.com/in/marilynmalpu">LinkedIn</a>
         </li>
       </ul>
+    </section id="message_from_section">
+    <h2>Leave a message</h2>
+    <form name="leave_message" class="message-form">
+      <label for="usersName">Name:</label>
+      <input
+      type="text"
+      id="usersName"
+      name="usersName"
+      required="true"
+      /><br />
+      <label for="usersEmail">Email:</label>
+      <input
+      type="email"
+      id="usersEmail"
+      name="usersEmail"
+      required="true"
+      /><br />
+      <label for="usersMessage">Message:</label>
+      <textarea
+      name="usersMessage"
+      id="usersMessage"
+      required="true"
```

Comment:
For boolean attributes, we don't pass any values. We just pass its name if we want it to be `true` and omit it if we want it to be false.
```suggestion
      required
```